tests/unit/api/test_offline_ingest_query_graph.py

```diff
@@ -710,7 +732,7 @@ def test_databricks_specs(
     # Each FeatureLookup or FeatureFunction object defines a set of features to be included
     features = [
         FeatureLookup(
-            table_name="fb_entity_cust_id_fjs_1800_300_600_ttl_[CATALOG_ID]",
+            table_name="feature_engineering.some_schema.fb_entity_cust_id_fjs_1800_300_600_ttl_[CATALOG_ID]",
```

Comment:
Fully qualified table & function names are expected here. When i'm trying to add backtick to quoted the name like ``feature_engineering`.`some_schema`.fb_entity_cust_id_fjs_1800_300_600_ttl_[CATALOG_ID]`, it fails due to not able to find the table/function.

Reply:
Might want to check if this also works if there are unicode characters.

Reply:
I tried to add some space in the schema name, it fails when generating the feature store.

Reply:
Let's say if it does not work for unicode characters, do you know any other way to handle this? (since backticks does not work in this case)

Reply:
I'm not sure, backtick was the way to go to handle special characters.